app-blank-after-kendo-upgrade.md

```diff
@@ -0,0 +1,43 @@
+
+---
+title: Upgrading from Kendo UI Core to Professional Breaks App
+description: After upgrading an app through the Package Manager to use Kendo UI Professional it appears blank on start up. 
+type: troubleshooting
+page_title: Upgrading from Kendo UI Core to Professional Breaks App
+slug: app-broken-after-kendo-upgrade
+position: 0
```

Comment:
Not sure if this will have negative effect after build, but let's keep the position field blank instead of adding 0s.